SkietbaanBE/SkietbaanBE/Controllers/CompetitionController.cs

```diff
@@ -0,0 +1,129 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using SkietbaanBE.Models;
+
+namespace SkietbaanBE.Controllers
+{
+    [Produces("application/json")]
+    [Route("api/Competition")]
+
+    public class CompetitionController : Controller
+    {
+        private ModelsContext _context;
+        public CompetitionController(ModelsContext db)
+        {
+            _context = db;
+        }
+ //----------------------------------------------------------------------------------------------------------------------------------------
+        /** The method to return an array of competition objects that hold the true Status*/
+        // GET: api/Competition
+        [HttpGet]
+        public IEnumerable<Competition> Get()
+        {
+            //get the competitions where(Status == true)
+            var competitionIDsQuery = from Comp in _context.Competitions
```

Comment:
You use LINQ queries in the rest of the project?  Why not here?

Reply:
Other queries are easier in sql, then LINQ